### DISCUSSION TITLE
BasicVSR++ Denoise Train Error

### DISCUSSION THREAD

**Discussion by mikolez** [open-mmlab/mmediting#1289] [Datasets] [answered]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmediting/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmediting).

### 🐞 Describe the bug

When I try to run the training for denoising using the suggested script and configuration file, I get "ValueError: The input sequence is not long enough to support the current choice of [interval] or [num_input_frames]" at BasicVSR_PlusPlus/mmedit/datasets/pipelines/augmentations.py, line 1053. I investigated the code a bit and it seems that the problem is with the way data is loaded. It looks like there is a variable at some point of loading the data which is called sequence_length and it is always zero because it calculates number of png images. However, DAVIS dataset only consists of jpg images, so that's why the sequence_length is zero. I assume the data is preprocessed for denoising task in some way and saved in png format in the end. The data preprocessing step is unfortunately missing in this repo, so I am wondering could it be the reason for my error? Or there is no preprocessing when training for denoising? I am trying to run the training with the DAVIS dataset.

Curious if you have any clue why my error is occurring.

Thank you!

### Environment

The bug is not related to the environment because the inference is running well.

**Comment by Z-Fran** [comment, marked as answer]
We set **filename template** to load datasets, like BasicVSR_PlusPlus/mmedit/datasets/pipelines/augmentations.py, line 1074. I think you can set this parameter to meet your need, like `filename_tmpl='{:08d}.jpg'`
